Source/Core/Jobs/EventPostsJob.cs

```diff
@@ -89,9 +89,10 @@ public class EventPostsJob : QueueJobBase<EventPost> {
 
                 // Discard any events over there limit.
                 events = events.Take(eventsToProcess).ToList();
-
-                // Increment by count - 1 since we already incremented it by 1 in the OverageHandler.
-                await _organizationRepository.IncrementUsageAsync(project.OrganizationId, false, events.Count - 1, applyHourlyLimit: false).AnyContext();
+                
+                // Increment the count if greater than 1, since we already incremented it by 1 in the OverageHandler.
+                if (events.Count > 1)
+                    await _organizationRepository.IncrementUsageAsync(project.OrganizationId, false, events.Count - 1, applyHourlyLimit: false).AnyContext();
```

Comment:
@ejsmith I'm wondering if we should just be incrementing the usage by the total number of events - 1 before we do the take. Otherwise we won't know the true number of events submitted.